WireDomain/Sources/WireDomain/Repositories/Message/MessageLocalStore.swift

```diff
@@ -317,11 +319,17 @@ public final class MessageLocalStore: MessageLocalStoreProtocol {
         date: Date
     ) async throws -> (ZMOTRMessage, isNew: Bool) {
         try await context.perform { [self] in
-            guard let clearedTime = conversation.clearedTimeStamp,
```

Comment:
There was a bug here... we were trying to unwrap the cleared timestamp and otherwise throwing an error, but not all conversations have a cleared time stamp which meant that the error was thrown for all messages. Instead, we check if there is a cleared timestamp, then the incoming message is after that time, otherwise thrown an error.